starter_code/elevator.js

```diff
@@ -1,19 +1,44 @@
 class Elevator {
   constructor(){
-    this.floor      = 0;
+    this.floor = 0;
+    this.list_floor = [0];
     this.MAXFLOOR   = 10;
-    this.requests   = [];
+    this.requests   = ["up","down"];
   }
 
-  start() { }
-  stop() { }
-  update() { }
+  start() {
+    setInterval(this.update, 2000);
+   }
```

Comment:
Guarda el Intervalo en una propiedad ej. this.interval para hacer el clear en stop()